packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -342,13 +347,16 @@ function updateMemoComponent(
     // Default to shallow comparison
     let compare = Component.compare;
     compare = compare !== null ? compare : shallowEqual;
+    pauseHooks(true);
```

Comment:
I think the test passes without these calls. Memo doesn't call "prepare to use hooks" so we're not in an area where there's a current fiber. You should be able to just remove them.

Reply:
oh interesting, you're right they pass. I recollect how I broke this - I had a bad test, and 'fixed' it after I had my code in place. bleurgh, thanks for the catch.